File: CB.EN.U4CYS22057/30_03_2023/python/array.py
# Declaring array
array = [0] * (10)

# Declaring variables
# For loop for taking input
for x in range(0, 9 + 1, 1):
    print("Enter " + str(x + 1) + " integer value")
    input = int(input())
    
    # assigning values in array
    array[x] = input

# For loop for writing output
for x in range(0, 9 + 1, 1):
    print("The " + str(x + 1) + " value in array is " + str(array[x]))

File: CB.EN.U4CYS22057/30_03_2023/c/array.c
#include <stdio.h>

int main() {
    // Declaring array
    int array[10] = {0};

    // For loop for taking input
    for (int x = 0; x < 10; x++) {
        printf("Enter %d integer value:\n", x+1);
        int input;
        scanf("%d", &input);

        // assigning values in array
        array[x] = input;
    }

    // For loop for writing output
    for (int x = 0; x < 10; x++) {
        printf("The %d value in array is %d\n", x+1, array[x]);
    }

    return 0;
}